WEBPACK_UPDATE.md

```diff
@@ -0,0 +1,59 @@
+# Upgrading from angular-cli@1.0.0-beta.10 to the angular-cli@1.0.0-beta.11-webpack 
+
+To update `angular-cli` to the webpack preview, you must update both the global package, project's local package and your project files.
+
+## Global package:
+```
+npm uninstall -g angular-cli
+npm cache clean
+npm install -g angular-cli@1.0.0-beta.11-webpack
+```
+
+## Local project package:
+```
+rm -rf node_modules dist tmp typings
+npm install --save-dev angular-cli@1.0.0-beta.11-webpack
+```
+
+IMPORTANT NOTE: 
+Currently project generated with `ng new` will use a wrong CLI version (see https://github.com/angular/angular-cli/issues/1528). After initializing your project, run `npm install --save-dev angular-cli@1.0.0-beta.11-webpack` to set the correct version.
+
+## Project files
+
+You will need to run `ng init` to check for changes in all the auto-generated files created by `ng new` and allow you to update yours. You are offered four choices for each changed file: `y` (overwrite), `n` (don't overwrite), `d` (show diff between your file and the updated file) and `h` (help).
```

Comment:
Please remove `ng init`. It's dangerous that people are relying on it; it will not touch files that have been deleted, or files added by the user. We need to list a higher level list of things to take care when updating the application.